lib/hanami/model.rb

```diff
@@ -73,6 +73,7 @@ def self.container
 
     # @since 0.1.0
     def self.load!(&blk) # rubocop:disable Metrics/AbcSize
+      configuration.gateway.use_logger(configuration.logger)       unless configuration.logger.nil?
```

Comment:
I see.

Here is a benchmark, so we can make informed decisions:-)

> Comparison:
no logger, nil check:  8778993.1 i/s
   null logger, call:  6309574.3 i/s - 1.39x slower

```ruby
require 'benchmark/ips'


class NullLogger
  def info(message)
  end
end

class Something
  def initialize(logger)
    @logger = logger
  end

  def work_with_null_check
    @logger.info('working hard') if @logger
  end

  def work_without_null_check
    @logger.info('working hard')
  end
end

null_logger = Something.new(NullLogger.new)
no_logger = Something.new(nil)

Benchmark.ips do |x|
  x.compare!
  
  x.report('null logger, call') do
    null_logger.work_without_null_check
  end

  x.report('no logger, nil check') do
    no_logger.work_with_null_check
  end
  
end

```